docs/contribute/testing/best-practises copy.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
First of all, the file name is not correct.

Second, I don't think we need to cover testing wasm apps.